README.md

```diff
@@ -184,6 +184,24 @@ If the execution is specified (does not run by default), the `containerize-depen
 	</plugin>
 ```
 
+
+To update the Dockerfile with containerize dependencies logic, you need to specify the Dockerfile location by adding the `dockerfile` tag under `configuration` and, also, to insert both the `#HABUSHU_BUILDER_STAGE` and `#HABUSHU_FINAL_STAGE` tags in the Dockerfile in the preferred location. 
+Note: Miss adding the tags in the Dockerfile or configuring the dockerfile location will result the Dockerfile not being updated.
+
+`#HABUSHU_BUILDER_STAGE` tag will be replaced by the habushu builder stage logic to, first, copy the generated dependencies files onto a Docker container and then to create the virtual environment for building the single habushu module.
+
+`#HABUSHU_FINAL_STAGE` tag will be replaced by the habushu final stage logic to copy over the built wheel file to the final stage.
+```dockerfile
+#HABUSHU_BUILDER_STAGE
+
+FROM redhat/ubi9-minimal:latest
+RUN microdnf install -y python3.11
+
+#HABUSHU_FINAL_STAGE
+
+ENTRYPOINT ["/venv/bin/python3.11", "-m", "pets.main", "--enable_docs_url", "True"]
+```
+
```

Comment:
C: we need to add the new configuration of `dockerfile` path to this README, preferably at the bottom of the configurations section

Reply:
the configuration is added to the readme. I can move it down to be more obvious.

Reply:
We have a separate section that lists out all of the available configurations. We need to add a new section there for the dockerfile config